### ISSUE TITLE
Idea: take advantage of 'sidebars'

### ISSUE THREAD

**Issue by jimmcnulty41** [wikitree-website/wikitree-page#3]
Some wikipedia articles (such as https://en.wikipedia.org/wiki/Programming_paradigm) have a sidebar that summarizes the topical area that the article 'fits into'.

In cases like these, where we have a structure between pages that is "known", we could use that to provide an embedded view (see #2 )

**Comment by alexburner** [comment]
True! Semi-related, Wikipedia also has article preview popovers on link hover now (https://www.mediawiki.org/wiki/Page_Previews) which would be cool to include

One solution would be swapping all of Wikitree from a destination site (with a nested cannibalized Wikipedia iframe) to a browser extension that runs while visiting the actual Wikipedia site
